.github/workflows/ci.yml

```diff
@@ -45,6 +45,8 @@ jobs:
 
       - name: Check
         run: ./gradlew $GRADLE_ARGS check jacocoTestReport
+             -Pnpm.publish.repository.github.authToken=batman
```

Comment:
nitpick

I do love having little "easter eggs" or fun commentary in code, but I worry that `batman` isn't immediately obvious to be a placeholder to someone who later joins the team and is diving into this code.

I'd suggest either adding a fun comment or updating the token to be clearer of intent.

```yml
      - name: Check
        # Using placeholder auth token and version for Gradle tasks that require its presence but don't use it.
        run: ./gradlew $GRADLE_ARGS check jacocoTestReport
             -Pnpm.publish.repository.github.authToken=batman
```

or...

```yml
      - name: Check
        # "Come on, Robin, to the Batcave. We haven't one moment to lose!"
        # Using placeholder auth token and version for Gradle tasks that require its presence but don't use it.
        run: ./gradlew $GRADLE_ARGS check jacocoTestReport
             -Pnpm.publish.repository.github.authToken=api-test-auth-token-placeholder
```

Reply:
There is an alternative. The token can be specified in the Gradle task directly inside the `repository("github") { ... }` block: `authToken = "notarealtoken"`.

This will fail if you try to publish to GitHub for real unless it is overridden from the command line but that is fine as the CI task is the only place it happens and it does exactly that.

Given how much friction there is in having to supply a dummy token (and even having to supply it in order to see the tasks in the `./gradlew tasks` list) this may be a preferred solution.

Reply:
Oh, yes, I prefer @davidtaylor-juul's solution.